Author: José Manuel Santamaría Lema <panfaust@gmail.com>
Description: This patch removes the nlist.h include for [k]FreeBSD systems.
 This include is making kde-workspace FTBFS on gnu/kfreebsd while it isn't
 actually used.
--- a/ksysguard/ksysguardd/FreeBSD/cpuinfo.c
+++ b/ksysguard/ksysguardd/FreeBSD/cpuinfo.c
@@ -27,7 +27,6 @@
 
 #include <devstat.h>
 #include <fcntl.h>
-#include <nlist.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
